src/test/integration/intercom/test_backend_scheduler.py

```diff
@@ -54,21 +50,17 @@ def update_analysis_of_single_object(self, fw):
 
 @pytest.fixture(name='intercom')
 def get_intercom_for_testing():
-    with TemporaryDirectory(prefix='fact_test_') as tmp_dir:
```

Comment:
>  don't you use pre-commit?

It fails so often for unrelated changes that I developed the bad habit to commit without verification.
Also I just recently realised that the pre-commit hooks don't run wen using `git rebase --continue`.
Sorry for the noise.

Reply:
> It fails so often for unrelated changes that I developed the bad habit to commit without verification most of the time.

Maybe we should fix some of the problems (or leave "ignore comments" where appropriate) so that it gets better  :sweat_smile:

Reply:
We definitly should. For this PR I'll try not forget to run it.